types/fast-text-encoding/fast-text-encoding-tests.ts

```diff
@@ -1,5 +1,9 @@
 import 'fast-text-encoding';
```

Comment:
This doesn't appear to have any effect.

Reply:
Ah, I didn't realise that you access the namespace without importing, thanks!